src/Loop.php

```diff
@@ -59,155 +59,162 @@ public static function stop()
     /**
      * Defer the execution of a callback.
      *
-     * @param callable $callback The callback to defer.
+     * @param callable(mixed $data, string $watcherIdentifier) $callback The callback to defer.
```

Comment:
@bwoebi @trowski We don't pass `$stream` as the first parameter, we pass it as `$cbData` as it seems: http://amphp.org/docs/amp/reactor-control.html#onreadable